internals/utils/objectContext.go

```diff
@@ -210,7 +227,7 @@ func (r *ObjectContext) UpdateNsByparent(nsparent *ObjectContext, nsChild *Objec
 		labels[danav1.Aggragator+nsName] = "true"
 		labels[danav1.Parent] = nsparent.Object.(*corev1.Namespace).Name
 		labels[danav1.Hns] = "true"
-		labels[danav1.ResourcePool] = GetNamespaceResourcePooled(nsChild)
+		//labels[danav1.ResourcePool] = GetNamespaceResourcePooled(nsChild)
```

Comment:
Maybe we can delete this all together? Don't think we use the namespace ResourcePool label anywhere else